### PR TITLE
Add action to ensure dist is up to date

### PR DESCRIPTION
**Pull request by dyladan**
image

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -200,6 +200,6 @@ export async function loadYaml(client: Client, ref: string, location: string): P
         const contents = await getFileContents(client, ref, location);
         return yaml.load(contents, { filename: location });
     } catch (err) {
-        throw new Error(`Failed to load configuration ${ref.slice(0, 7)} ${err.message} ${location}`)
+        throw new Error(`Failed to load configuration ${ref.slice(0, 7)} ${err} ${location}`)
```
Comment (dyladan) [inline src/utils.ts:203]: Required to fix the build with latest typescript

**Comment by trask** [comment]
can we wait to merge this until after the cherry-picks are merged? (#15)

File: package.json [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
-    "typescript": "^4.3.5"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
     "@actions/core": "^1.4.0",
     "@actions/github": "^5.0.0",
-    "fp-ts": "^2.11.1",
-    "io-ts": "^2.2.16",
+    "fp-ts": "^2.13.1",
+    "io-ts": "^2.2.20",
```
Comment (dyladan) [inline package.json:31]: Some of the dependency updates aren't specifically needed but I think they're all a good idea

**Comment by dyladan** [comment]
> can we wait to merge this until after the cherry-picks are merged? (#15)

sure

File: .github/workflows/build-dist.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Ensure Dist Build
+on:
+  push:
```
Comment (trask) [inline .github/workflows/build-dist.yml:3]: (e.g. so it won't apply on dependabot branches)

```suggestion
  push:
    branches:
      - main
```

**Review by trask — APPROVED** [review]

File: src/main.ts [hunk 4]
```diff
@@ -22,6 +22,8 @@ async function main() {
 
     const author = await getPullAuthor(client);
 
+    console.log('remove me')
+
```
[the remark this answers is not in the document]
Comment (dyladan) [reply src/main.ts:26]: yes

**Pull request merged**